scripts/build.py

```diff
@@ -140,17 +140,19 @@
 
 _PARSER.add_argument(
     '--prod_env', action='store_true', default=False, dest='prod_mode')
+_PARSER.add_argument(
+    '--deploy_mode', action='store_true', default=False, dest='deploy_mode')
```

Comment:
I'm not sure whether we should have two modes in this script, considering prod_mode is not used as a mode in this script, right?

The line which calls this script has bot `--prod_env` and `deploy_mode` flags and this script it means it's in both modes. I would suggest changing the prod_mode to prod_env (Considering it's not working as a mode anymore, right?)

Reply:
Done.